### PR TITLE
RATEST-69: Fix ignored conditions steps

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [id](https://issues.openmrs.org/projects/RATEST/issues/RATEST-69?filter=allopenissues)
Refactored @ignore annotation on scenarios that were being skipped during automation

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@kdaud you can actually add a little description about your changes in the pr description section

**Comment by kdaud** [comment]
> add a little description about your changes in the pr description section

Hey @HerbertYiga, have addressed this concern

File: src/test/resources/features/13-condition/conditions.feature [hunk 1]
```diff
@@ -18,31 +18,31 @@ Feature: Patient's Conditions Managment
...
 
-  @ignore
+  @run
   Scenario: Set first condition to active
     And User clicks on set active button
     Then System should move condition to active section
     And Quit browser
 
-  @ignore
+  @run
```
Comment (kaweesi) [inline src/test/resources/features/13-condition/conditions.feature:33]: these can only be enabled in the same change that introduces their respective step descriptions

**Comment by sherrif10** [comment]
@kdaud  did you merge conflicts

**Comment by HerbertYiga** [comment]
> @kdaud did you merge conflicts

you have some conflicting files

**Comment by kaweesi** [comment]
i will close this for now until we can have the steps written

**Pull request closed without merging**